RFTrain/src/tester.py

```diff
@@ -0,0 +1,13 @@
+def train_by_config(config_path):
+    config = import_train_configuration(config_file=config_path)
+    dataset_importer = DatasetImporter(config)
+
+    if config.get('classifier_type').lower() == 'k-dnn':

```

Comment:
why we pass if a classifier type is equal to 'k-dnn'?

Reply:
remnants of old, this feature should not be in the framework at all